server/middlelayer-interaction.es6

```diff
@@ -1,51 +0,0 @@
-'use strict';
```

Comment:
Has this now been moved to the individual permit types? Or this is now just moving to the next sprint.